README.md

```diff
@@ -98,8 +97,6 @@ It supports the following flags:
   `foo=bar/baz.go`, where `bar/baz.go` is the source file and `foo` is the
   package name of that file used by the -source file.
 
-- `-build_flags`: (reflect mode only) Flags passed verbatim to `go build`.
-
```

Comment:
Should we keep these lines now since we are keeping the flag?

Reply:
You're right, i forgot to do this